src/main/webapp/components/workspace/workspace.js

```diff
@@ -71,7 +71,7 @@ export default () => {
 
   const [listResults, setListResults] = React.useState([])
   const [currentQuery, setCurrentQuery] = useState(null)
-  const [queries, setQueries] = useState([])
+  const [queries, setQueries] = useState()
```

Comment:
![image](https://user-images.githubusercontent.com/4467636/77018552-20a6a300-693b-11ea-90dd-aae3308fb1f1.png)
Having an initial state of undefined prevents this^ from showing before queries are fetched on a workspace that has existing queries.